public/app/core/logs_model.ts

```diff
@@ -480,3 +466,48 @@ function getIdField(fieldCache: FieldCache): FieldWithIndex | undefined {
   }
   return undefined;
 }
+
+// Format timeSpan (in sec) to string used in log's meta info
+function formatTimeSpanToTimeString(timeSpanSec: number): string {
```

Comment:
It looks like more accurate version of [secondsToHms()](https://github.com/grafana/grafana/blob/master/packages/grafana-data/src/datetime/rangeutil.ts#L237). Would it make sense to move it to [rangeutil.ts](https://github.com/grafana/grafana/blob/master/packages/grafana-data/src/datetime/rangeutil.ts)?

Reply:
Yes!

Reply:
Moved.